app/controllers/root_controller.rb

```diff
@@ -109,12 +110,58 @@ def publication
       end
 
       @interaction_details = prepare_interaction_details(@publication, authority_slug, snac)
+    elsif @publication.format == 'local_transaction'
+      @location = fetch_location @postcode
+
+      if @location.nil?
+        @postcode_error = "fullPostcodeNoMapitMatch"
```

Comment:
This conversation happened over [here](https://github.com/alphagov/gds-api-adapters/pull/378).